aws/src/main/python/beam_lambda/lambda_function.py

```diff
@@ -265,6 +265,33 @@
                   'z1d.large', 'z1d.xlarge', 'z1d.2xlarge', 'z1d.3xlarge', 'z1d.6xlarge', 'z1d.12xlarge',
                   'x2gd.metal', 'x2gd.16xlarge']
 
+instance_types_with_memory = {
```

Comment:
Could we rename to either `instance_type_to_memory` or `memory_in` (that way you could write `memory_in[instance_type]`)

Reply:
applied this suggestion